code/drasil-code/lib/Language/Drasil/Code/Imperative/Parameters.hs

```diff
@@ -45,17 +44,12 @@ getInConstructorParams = do
   ps <- getParams cname In $ getCParams (cname `elem` defList g)
   return $ filter ((Just cname /=) . flip Map.lookup (clsMap g) . codeName) ps
 
--- | The inputs to the function for reading inputs are the input file name, and
--- the 'inParams' object if inputs are bundled and input components are separated.
--- The latter is needed because we want to populate the object through state
--- transitions, not by returning it.
+-- | The inputs to the function for reading inputs are the input file name.
 getInputFormatIns :: GenState [CodeVarChunk]
 getInputFormatIns = do
-  g <- get
-  let getIns :: Structure -> InputModule -> [CodeVarChunk]
-      getIns Bundled Separated = [quantvar inParams]
-      getIns _ _ = []
-  getParams "get_input" In $ quantvar inFileName : getIns (inStruct g) (inMod g)
+  let getIns :: [CodeVarChunk]
+      getIns = []
+  getParams "get_input" In $ quantvar inFileName : getIns
```

Comment:
Is there a reason we can't do this? Or this is "scaffolding" for a later change?
```suggestion
  getParams "get_input" In $ quantvar inFileName : []
```

Reply:
Oh true, that's from the fix I made a minute ago.  I tried simplifying that more, but Haskell couldn't figure out its type.  There might be a better way to do it than where I left it, though.  I'll take a look.

Reply:
I think you can do `([] :: [CodeVarChunk])` to specify the type of an empty list

Reply:
Turns out I was just messing up the syntax, and we don't even need the `[]`!